lib/usb/usb_dwc_common.c

```diff
@@ -214,11 +214,11 @@ uint16_t dwc_ep_write_packet(usbd_device *usbd_dev, uint8_t addr,
 
 	/* Copy buffer to endpoint FIFO, note - memcpy does not work.
 	 * ARMv7M supports non-word-aligned accesses, ARMv6M does not. */
```

Comment:
_if_ you're going to to do this sort of thing, you need to update the comments to.